lib/device/lib/marionette.js

```diff
@@ -53,7 +53,8 @@ var Marionette = function(device) {
  * @returns {Promise.<MarionetteClient.Client>}
  */
 Marionette.prototype.startSession = function() {
-  debug('[Creating driver] %s:%d connectionTimeout: %d', this.host, this.port, this.timeout);
+  debug('[Creating driver] %s:%d with %dms connection timeout',
```

Comment:
Yeah I didn't realize it until Fabrice thought there was an error, so I just changed it. :P